Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/firebase/FirebaseHelper.kt

```diff
@@ -26,6 +27,7 @@ class FirebaseHelper private constructor() {
         this.aboutDatabase = mainDatabase.child("about")
         this.faqDatabase = mainDatabase.child("faq")
         this.cocDatabase = mainDatabase.child("conductCode")
+        this.userDatabase = mainDatabase.child("user")
```

Comment:
nit: remove the `this`, it's not needed and so java :)

Reply:
Could these assignments in the init block be inlined into the declaration of each val?

Reply:
Removed `this` :)

@ericmaxwell2003 Unfortunately, all of these calls need to be made before disk persistence is enabled. Inlining the assignments produces an error:

```
Caused by: com.google.firebase.database.DatabaseException: Calls to setPersistenceEnabled() must be made before any other usage of FirebaseDatabase instance.
```